src/Components/CandidateTable/fakeData.ts

```diff
@@ -209,15 +187,15 @@ export const fakeUserData: User[] = [
   },
   {
     id: 10,
-    hashName: "3ff6d51a706a176cd3504635462e655d84cb12321d03b3a0a4f184c4245fe89a",
+    hash: "3ff6d51a706a176cd3504635462e655d84cb12321d03b3a0a4f184c4245fe89a",
 
     nick: "Likence",
     avatarImage: "https://www.publicdomainpictures.net/pictures/280000/nahled/not-found-image-15383864787lu.jpg",
```

Comment:
https://via.placeholder.com/300.png/09f/fff BTW. Again, don't change, just use it when you need placeholders moving forward.